client/plots/barchart.js

```diff
@@ -128,19 +128,23 @@ class Barchart {
 					]
 				},
 				{
-					label: 'Color bars',
+					label: 'Multicolor bars',
+					title: 'Color bars ussign preassigned colors from the dataset or a random color',
```

Comment:
Typo on `ussign'. But also, if there are no preassigned colors, then it will use the `defaultColor`, not a random color